src/mainwindow.cpp

```diff
@@ -308,12 +308,12 @@ void MainWindow::showConsoleContextMenu(const QPoint &pt)
 
 void MainWindow::closeEvent(QCloseEvent *event)
 {
-    QMessageBox::StandardButton ret = QMessageBox::question(this, "Iaito",
+    QMessageBox::StandardButton ret = QMessageBox::question(this, "iaito",
```

Comment:
I would prefer to keep "Iaito" instead of "iaito" for the window/dialog titles :)